sale_input_barcode/tests/test_.py

```diff
@@ -0,0 +1,24 @@
+# © 2022 David BEAL @ Akretion
```

Comment:
It's quite uncommon to have that file naming.

See here for convention : https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id12

Reply:
While there is only one file included test in 1 module, it's more readable to avoid to have module name in test file name

`openerp_test odoo.addons.sale_input_barcode.tests.test_: Starting Test.test_input_line ...`

vs

`openerp_test odoo.addons.sale_input_barcode.tests.test_sale_input_barcode: Starting Test.test_input_line`

In my opinion the first one is more readable avoiding duplicated information and could be the default one.
Not sure we have to trigger a long debate on this subject. I'm declared as maintainer on this module, I'll maintain it.

Reply:
Not to have a long debate about it and no need to change. But explicit things are better IMHO

Reply:
Hello.
well regarding that minor topic, at oca we generally name the file ``test_module.py``.

- https://github.com/OCA/pos/commits/12.0/pos_order_return_traceability/tests
- https://github.com/OCA/pos/tree/12.0/pos_order_mgmt/tests

no occurences of ``test_.py``.

Reply:
test_module seems fine to me.

Reply:
I've applied this name